integration/oneup_custom_test.go

```diff
@@ -0,0 +1,10 @@
+//+build integration
+
+package integration
+
+import "testing"
+
+func TestOneUpCustom(t *testing.T) {
+	f := newFixture(t, "oneup_custom")
```

Comment:
could you also add a CurlUntil like in the other integration tests to ensure the server comes up?

Reply:
Whoops, I added that to the wrong file locally and didn't commit it. 😬